python/ray/_private/runtime_env/packaging.py

```diff
@@ -784,7 +785,20 @@ async def download_and_unpack_package(
                             "`pip install boto3` to fetch URIs in s3 "
                             "bucket. " + install_warning
                         )
-                    tp = {"client": boto3.client("s3")}
+                    s3_kwargs = {}
```

Comment:
Hey @laysfire  , not a ray maintainer but I will suggest adding a test case for your change.